woocommerce/woocommerce-ios
# Order list: implemented the Order Filters screen without actions

Part of #5243 

## Description
In this PR, I implemented the screen for showing the first filters (Order Status and Date Range) that will be used for filtering the Order List. I implemented `FilterOrderListViewModel` which conforms to `FilterListViewModel`. The idea is to reuse `FilterListViewController`, the same components used in Products list for filtering the results.

## Testing
1. Launch the app it in debug (because of the feature flag).
2. Open the order list.
3. You should be able to tap the `Filter` button.
4. You should be able to view the `Filters` screen with two options: Order Status and Date Range.
5. Under Order Status, you should be able to select just one filter.
6. Under Date Range, you should be able to select just one filter (for the moment the custom range option is one of the selectable options. In a future PR, it will become an action).
7. After selecting at least an option, you should see in the Filters screen the "Clear all" button. Pressing it, should clear the filters.
8. Press Show Orders button. The view should be dismissed (no filters will be applied to order list for the moment. The functionality will be implemented in future PRs).

## Screenshots
| Filters screen            |  Order Status filters | Date Range filters |
:-------------------------:|:-------------------------:|:-------------------------:
![Simulator Screen Shot - iPhone 13 Pro - 2021-10-22 at 14 51 17](https://user-images.githubusercontent.com/495617/138459739-fd86391e-73a8-4c64-b6ba-251f6c04d1f4.png) | ![Simulator Screen Shot - iPhone 13 Pro - 2021-10-22 at 14 51 24](https://user-images.githubusercontent.com/495617/138459774-1f9e14bd-0b4a-45a3-81c7-fd5edb0d8286.png) | ![Simulator Screen Shot - iPhone 13 Pro - 2021-10-22 at 14 51 28](https://user-images.githubusercontent.com/495617/138459814-6a2a20f3-d3d4-4171-9228-b0d4d5ac152e.png)

Update release notes:
